### PR TITLE
[9.0][FIX]Update README.md

### PR DESCRIPTION
**Pull request by elicoidal**
Added Badges and default introduction

### CODE REVIEW

**Comment by elicoidal** [comment]
I am not sure about coverall and how to get the list of addons updated in the README

**Comment by pedrobaeza** [comment]
The same about codecov.

**Comment by elicoidal** [comment]
@pedrobaeza

**Comment by astirpe** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
+[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/222/9.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-business-requirement-222)
 [![Build Status](https://travis-ci.org/OCA/business-requirement.svg?branch=9.0)](https://travis-ci.org/OCA/business-requirement)
+[![codecov](https://codecov.io/gh/OCA/business-requirement/branch/9.0/graph/badge.svg)](https://codecov.io/gh/OCA/project)
```
Comment (astirpe) [inline README.md]: Must be `business-requirement` instead of `project`

**Review by astirpe — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
Of course!

**Review by astirpe — APPROVED** [review]

**Review by sbidoul — CHANGES_REQUESTED** [review]
The addons placeholder is incorrect.

**Comment by elicoidal** [comment]
@sbidoul 
> The addons placeholder is incorrect.

Can you precise as I dont know the rule?  Should I remove it? How to get the addons list properly displayed? see: #75

**Comment by elicoidal** [comment]
> Can you precise as I dont know the rule? Should I remove it? How to get the addons list properly displayed? see: #75

Saw the answer later, sorry for the noise.
@sbidoul

**Review by dreispt — APPROVED** [review]

**Pull request merged**